models/breast_density_classification/scripts/script.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
Hi @vikashg , thanks for making this new PR.
Here are some code format issues (https://github.com/Project-MONAI/model-zoo/actions/runs/3209554960/jobs/5246367812#step:7:73)

Reply:
I pushed some commits and the format issue has been resolved.